coredns/tests/test_check.py

```diff
@@ -0,0 +1,67 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.coredns import CoreDNSCheck
+
+import mock
+import os
+import pytest
+
+instance = {
+    'prometheus_endpoint': 'http://localhost:9153/metrics',
+}
+
+# Constants
+CHECK_NAME = 'coredns'
+NAMESPACE = 'coredns'
+
+
+@pytest.fixture
+def aggregator():
```

Comment:
You can remove this function since it is part of `datadog_check_base`. 
You can look at the `Vault` integrations for an example.